src/helpers/Auth.js

```diff
@@ -0,0 +1,17 @@
+import jwt from 'jsonwebtoken';
+import { SECRET_KEY } from '../config/constants';
+/**
+ * Class handles JWT authentication
+ */
+class Auth {
```

Comment:
Instead of naming this Auth,js, having it as JWT.js and the class name should be JWT will be nice.